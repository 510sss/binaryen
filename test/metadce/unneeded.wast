(module
 (export "wasm_func_a" (func $a_wasm_func))

 (func $a_wasm_func (param $x i32)
  (block $nothing
   ;; do nothing interesting
   (nop)
   (local.set $x (i32.const 10))
   (nop)
   (return)
  )
 )
)

