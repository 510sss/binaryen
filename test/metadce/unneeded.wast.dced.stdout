unused: func$a_wasm_func$0
unused: wasm_export
